modules/python/Makefile.dist

```diff
@@ -1,8 +1,11 @@
-PYTHON_DIST_DIR=dist
+PYTHON_DIST_DIR?=dist
 
 .PHONY: python/dist
-python/dist: pipenv setup.py pipenv/generate_requirements
-	$(WITH_PIPENV) python setup.py sdist
+python/dist: pipenv setup.py requirements.txt requirements-dev.txt
+	$(WITH_PIPENV) python setup.py sdist --dist-dir $(PYTHON_DIST_DIR)
+
+	$(WITH_PIPENV) pip install -q --disable-pip-version-check wheel
+	$(WITH_PIPENV) python setup.py bdist_wheel --dist-dir $(PYTHON_DIST_DIR)
```

Comment:
Q. Should this be one target or split into two (`python/dist python/dist/wheel`)?

Reply:
Building wheels should probably be the default, but perhaps like this?
```Makefile
.PHONY: python/dist
python/dist: python/dist/src python/dist/wheel
```